Source/Core/Credit.js

```diff
@@ -30,19 +30,13 @@ define([
      * //Create a credit with a tooltip, image and link
      * var credit = new Cesium.Credit('Cesium', '/images/cesium_logo.png', 'http://cesiumjs.org/');
      */
-    function Credit(options, imageUrl, link) {
+    function Credit(options) {
         var text;
         var showOnScreen;
```

Comment:
@Jahrkor We can now declare these variables when they are assigned below. Other than that this all looks good!

Reply:
Thanks @ggetz